virt/node_maintenance/virt-automatic-certificates.adoc

```diff
@@ -0,0 +1,9 @@
+[id="virt-automatic-certificates"]
+= Automatic renewal of TLS certificates
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-automatic-certificates
+toc::[]
+
```

Comment:
The assembly needs an intro sentence. Maybe this would be a good place to say:
`All TLS certificates for {VirtProductName} components are renewed and rotated automatically. You are not required to refresh them manually.`

Reply:
Intro sentence was added to the assembly. Copied the text shown above.